test/ExportQueryExecutionTreeTest.cpp

```diff
@@ -636,6 +636,126 @@ TEST(ExportQueryExecutionTree, LiteralWithDatatype) {
   runConstructQueryTestCase(testCaseConstruct);
 }
 
+// ____________________________________________________________________________
+TEST(ExportQueryExecutionTree, TestIriWithEscapedIri) {
+  std::string kg =
+      "<s> <p> "
+      "<https://example.org/bim%20A%00c%03doof%11c%12testescape/%13test>";
+  std::string objectQuery =
+      "SELECT ?o WHERE { "
+      "BIND(IRI(<https://example.org/bim%20A%00c%03doof%11c%12testescape/"
+      "%13test>) AS ?o) }";
+  std::string expectedXml = makeXMLHeader({"o"}) +
```

Comment:
Those are not the escapes I meant, those change the meaning of the IRI.
I meant stuff like `<https:\\u0020example.org>`. etc. 
Also the `IRI` function in the bind can take a literal like `IRI("example\\n.org")`.

Let me know if you need help.

Reply:
I see that you implemented a separate test for the strings, but the first point remains valid.